persistence-test/src/main/java/io/stargate/it/PersistenceTest.java

```diff
@@ -611,32 +680,45 @@ public void testUDTWithAllSimpleTypes()
             .keyspace(ks.name())
             .addAllColumns(columns)
             .build();
-    dataStore.query().create().type(keyspace, udtType).execute();
+    dataStore.queryBuilder().create().type(keyspace, udtType).build().execute().join();
     ks = dataStore.schema().keyspace(ks.name());
 
     dataStore
-        .query()
+        .queryBuilder()
         .create()
         .table(ks.name(), table)
         .ifNotExists()
         .column("x", Int, PartitionKey)
         .column("udt", ks.userDefinedType(typeName).frozen())
-        .execute();
+        .build()
+        .execute()
+        .join();
 
     Object[] actualValues = new Object[columns.size()];
     for (int i = 0; i < columns.size(); i++) {
       actualValues[i] = typeToValue.get(columns.get(i).type());
     }
 
     dataStore
-        .query()
+        .queryBuilder()
         .insertInto(ks.name(), table)
         .value("x", 1)
         .value("udt", ks.userDefinedType(typeName).create(actualValues))
-        .execute();
+        .build()
+        .execute()
+        .join();
 
     java.util.List<Row> rows =
-        dataStore.query().select().star().from(ks.name(), table).where("x", Eq, 1).execute().rows();
+        dataStore
+            .queryBuilder()
+            .select()
+            .star()
+            .from(ks.name(), table)
+            .where("x", EQ, 1)
+            .build()
+            .execute()
+            .join()
+            .rows();
     assertThat(rows).isNotEmpty();
     // TODO: [doug] 2020-07-10, Fri, 16:43 we currently have a mismatch where some things are using
```

Comment:
Does this PR remove this issue?

Reply:
I don't think it's this issue that fixed it, but it does work now so added back.